unit_testing/test_labels.py

```diff
@@ -49,10 +57,7 @@ def fake_3dimage_sct():
     return img
 
 
-test_images = [fake_3dimage_sct(), fake_3dimage_sct2(), t2_img]
```

Comment:
What do you think about dropping this? I notice you copy-pasted this list instead of keeping it together. I didn't really think too hard why it might be one way or the other.